### DISCUSSION TITLE
Storybook integration

### DISCUSSION THREAD

**Discussion by penx** [allure-framework/allure2#2792] [Questions & Support]
Do you know if there's a way to get output from Storybook in to Allure?

I'm thinking for:
- [Component/interaction tests](https://storybook.js.org/docs/writing-tests/component-testing)  that use Storybook's play function
- Visual tests done via [Chromatic](https://www.chromatic.com/features/test), [Percy](https://www.browserstack.com/docs/percy/integrate/storybook) or [Applitools](https://applitools.com/solutions/storybook/) Storybook integrations
- [Accessibility tests](https://storybook.js.org/docs/writing-tests/accessibility-testing)
- Outputs from the [test runner](https://storybook.js.org/docs/writing-tests/test-runner)  (component tests as above, and ensuring stories without a play function render without error)

I see that the [Chromatic CLI can output a JUnit XML file](https://www.chromatic.com/docs/configure/#junitreport), though this is limited to testing via Chromatic, I'm not sure if it includes all of the test types listed above, and I'm not sure if I can go straight from a JUnit XML file to Allure.

**Comment by baev** [comment]
Since the test runner uses Jest, you can probably use [Allure Jest](https://allurereport.org/docs/jest/).

JUnit XML is also supported natively by Allure Report; for the generate command, specify the path to the directory with XML.
